docs/DeveloperGuide.md

```diff
@@ -1066,54 +1066,116 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
 
 ### 6.3 Use cases
 
-(For all use cases below, the **System** is `PatientSync` and the **Actor** is the `nurse`, unless specified otherwise)
+(For all use cases below, the **System** is `PatientSync` and the **Actor** is the `Nurse`, unless specified otherwise)
 
-**Use case: Add a patient**
+**Use case: UC01 - Add a patient**
 
 **MSS**
 
-1.  Nurse requests to add a patient
-2.  PatientSync adds the patient
+1. Nurse requests to add a patient.
+1. PatientSync adds the patient and display success message.
 
     Use case ends.
 
 **Extensions**
 
-* 2a. The information key in is wrong.
-    * 2a1. PatientSync shows an error message.
+* 1a. Nurse's input is invalid command format.
 
+    * 1a1. PatientSync shows an error message.
+           
       Use case ends.
+  
+* 1b. Nurse's input contains invalid parameters value. 
 
+    * 1b1. PatientSync shows an error message. 
```

Comment:
Perhaps can add one more use case where Nurse added patient with duplicated patient hospital ID?

Reply:
do you think it can be counted as invalid parameters value?

Reply:
ya I think should be can!